policy_check.go

```diff
@@ -82,34 +82,33 @@ type PolicyCheck struct {
 
 // PolicyActions represents the policy check actions.
 type PolicyActions struct {
-	IsOverridable bool `json:"is-overridable"`
+	IsOverridable bool `jsonapi:"attr,is-overridable"`
 }
 
 // PolicyPermissions represents the policy check permissions.
 type PolicyPermissions struct {
-	CanOverride bool `json:"can-override"`
+	CanOverride bool `jsonapi:"attr,can-override"`
 }
 
 // PolicyResult represents the complete policy check result,
 type PolicyResult struct {
-	AdvisoryFailed int  `json:"advisory-failed"`
-	Duration       int  `json:"duration"`
-	HardFailed     int  `json:"hard-failed"`
-	Passed         int  `json:"passed"`
-	Result         bool `json:"result"`
-	// Sentinel       *sentinel.EvalResult `json:"sentinel"`
-	SoftFailed  int `json:"soft-failed"`
-	TotalFailed int `json:"total-failed"`
+	AdvisoryFailed int  `jsonapi:"attr,advisory-failed"`
+	Duration       int  `jsonapi:"attr,duration"`
+	HardFailed     int  `jsonapi:"attr,hard-failed"`
+	Passed         int  `jsonapi:"attr,passed"`
+	Result         bool `jsonapi:"attr,result"`
+	SoftFailed     int  `jsonapi:"attr,soft-failed"`
+	TotalFailed    int  `jsonapi:"attr,total-failed"`
 }
 
 // PolicyStatusTimestamps holds the timestamps for individual policy check
 // statuses.
 type PolicyStatusTimestamps struct {
-	ErroredAt    time.Time `json:"errored-at"`
-	HardFailedAt time.Time `json:"hard-failed-at"`
-	PassedAt     time.Time `json:"passed-at"`
-	QueuedAt     time.Time `json:"queued-at"`
-	SoftFailedAt time.Time `json:"soft-failed-at"`
+	ErroredAt    string `jsonapi:"attr,errored-at"`
+	HardFailedAt string `jsonapi:"attr,hard-failed-at"`
+	PassedAt     string `jsonapi:"attr,passed-at"`
+	QueuedAt     string `jsonapi:"attr,queued-at"`
+	SoftFailedAt string `jsonapi:"attr,soft-failed-at"`
```

Comment:
This is fixed [here](https://github.com/hashicorp/jsonapi/pull/1), will update this comment soon.